### ISSUE TITLE
[SR-2111] Crash when using variable as a default value for method in class.

### ISSUE THREAD

**Issue by swift-ci** [swiftlang/llvm-project#4592]
|                  |                 |
   |------------------|-----------------|
   |Previous ID       | SR-2111      |
   |Radar             | rdar://problem/26978985         |
   |Original Reporter | njuri (JIRA User)      |
   |Type              | Bug    |
   |Status            | Closed        |
   |Resolution        | Done    |
        
<details>
  <summary>Environment</summary>

Xcode 8.0 beta 3  
OS X El Capitan 10.11.6

</details>

   <details>
  <summary>Additional Detail from JIRA</summary>

   |                  |                 |
   |------------------|-----------------|
   |Votes             | 0         |
   |Component/s       | Compiler, LLDB for Swift    |
   |Labels            | Bug, 3.0Regression, CompilerCrash        |
   |Assignee          | njuri (JIRA)      |
   |Priority          | Medium      |

   

   md5: f9cca2c1fae9d31635468825fa9df3eb

  </details>

**Issue Description:**

``` javascript

let a = 1

// Works ok
func foo(i : Int = a){
  print(i)
}

class TestClass {
  
  let b = 3
 
  // Crashing method
  func bar(j : Int = b){
    print(j)
  }
}
```

**Comment by rudkx** [comment]
Fixed in commit ee6c0a737b69853f3053aff34e8d6c8e8206a2b6.

**Comment by rudkx** [comment]
Can you confirm this is fixed, and close if so?
